README.md

```diff
@@ -1,2 +1,85 @@
-# plio-aws-discord-alerts
-AWS alerts directly to Discord!
+# Plio AWS Discord Alerts
+[![License: MIT](https://img.shields.io/github/license/avantifellows/plio-aws-discord-alerts?color=blue&style=flat-square)](LICENSE)
+![GitHub issues](https://img.shields.io/github/issues-raw/avantifellows/plio-aws-discord-alerts?style=flat-square)
+![Continuous Integration](https://img.shields.io/github/workflow/status/avantifellows/plio-aws-discord-alerts/Plio%20CI?label=Continuous%20Integration&style=flat-square)
+[![Discord](https://img.shields.io/discord/717975833226248303.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2&style=flat-square)](https://discord.gg/29qYD7fZtZ)
+
+This repository contains the Lambda function that ensures AWS alerts reaches to a Discord channel.
+
+## Installation
```

Comment:
shouldn't this section be under `docs/installation` with link in README as in other repos?